sdk/react/src/useDVCInitialized.ts

```diff
@@ -0,0 +1,24 @@
+import { useContext, useState } from 'react'
+import context from './context'
+
+export const useDVCInitialized = (): boolean => {
```

Comment:
Does this need to be awaitable? So that 2 modes are available to the user?

if awaited then the rendering is blocked

If not awaited then the app loads but the client is usable with default variables if not initialized

Reply:
That's what we used to have with `asyncWithDVCProvider` but I think we want to be able to use this hook like so:
```
export const MyApp = () => {
  const dvcReady = useDVCInitialized()

  if (!dvcReady) return <LoadingState/>

  return <TheRestofYourApp/>
}

Reply:
You could just use `withDVCProvider` wrapper on its own without this new `useDVCInitialized` hook!

Reply:
kk